### ISSUE TITLE
Unable to change the physical Keyboard layout

### ISSUE THREAD

**Issue by benintech** [waydroid/waydroid#599]
There has been an issue opened a year ago (https://github.com/waydroid/waydroid/issues/209), that has been closed without providing a clear answer. Is it possible to change the physical keyboard layout, or not ?

The layouts are stored under `/var/lib/waydroid/rootfs/system/usr/keylayout`, but `/var/lib/waydroid/rootfs` is a readonly filesystem, so unable to change that.

Is there a way to set up another physical keyboard layout, or is Waydroid a qwerty-only project ?

**Comment by benintech** [comment]
Closing it myself. You have to mount `system.img` read-write (see https://github.com/waydroid/waydroid/issues/603), and replace `system/usr/keylayout/Generic.kl` by one that matches your keyboard.

Note : soon you should be able to use overlays instead (see https://github.com/waydroid/waydroid/pull/580)

**Comment by notramo** [comment]
Please reopen it as the current situation is hugely suboptimal UX.

**Comment by rossmeier** [comment]
So to whom it may concern, I found a way to change the keyboard layout. By using a nested wayland compositor (which could even be a proxy that just fakes the keycodes) I was able to run waydroid with my native keyboard layout of the host. Maybe this idea could be built upon and integrated into waydroid in some way.
This is may sway config:
```
xwayland disable
for_window [app_id="Waydroid"] border none
focus_on_window_activation focus
exec "waydroid show-full-ui"
```
save as `waydroidconfig` and run using `sway -c waydroidconfig`
